src/js/components/cycleCount/CycleCount.jsx

```diff
@@ -95,7 +95,11 @@ const CycleCount = () => {
           filterParams={filterParams}
         />
         )}
-        {tab === TO_COUNT_TAB && <CycleCountToCount />}
+        {tab === TO_COUNT_TAB && (
+        <CycleCountToCount
+          filterParams={filterParams}
```

Comment:
are filters cleared when switching between tabs?

Reply:
no, and I thought it was the desired behavior, but I will ask for clarification

Reply:
Manon confirmed that it should be cleared